Gopkg.lock

```diff
@@ -1,27 +1,61 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
 
 
+[[projects]]
+  branch = "add-yarn-support"
```

Comment:
first merge `steps-ionic-archive` then update this dep to point to the master state of `steps-ionic-archive`